app/src/main/java/ch/epfl/sweng/rps/services/OfflineGameService.kt

```diff
@@ -55,8 +60,7 @@ class OfflineGameService(
 
     private suspend fun makeComputerMoves() {
         for (pc in computerPlayers) {
-//                uncomment when the proper tests with coroutines implemented
-//            delay(1_000)
+            delay(1_000)
```

Comment:
This should be customizable, i.e a parameter in OfflineGameService, or something similar.

Reply:
Now, I've pushed the specification of this param to the UI part (where it's default 1_000) so it can be in the future influenced by a user